crds/virtualimage.yaml

```diff
@@ -276,6 +305,7 @@ spec:
                     * Failed - There was a problem when creating a resource, details can be seen in `.status.failureReason` and `.status.failureMessage`.
```

Comment:
`, details can be seen in .status.failureReason` and `.status.failureMessage.` - remove this part of description